pages.pt_BR/common/certutil.md

```diff
@@ -0,0 +1,24 @@
+# certutil
+
+> Gerencie chaves e certificados em bancos de dados e tokens NSS
+> Mais informações: <https://manned.org/certutil>.
+
+- Criar um novo banco de dados de certificados:
+
+`certutil -N -d .`
+
+- Listar todos os certificados em um banco de dados:
+
+`certutil -L -d .`
+
+- Listar todas as chaves privadas em um banco de dados:
+
+`certutil -K -d . -f {{caminho/para/pwdfile.txt}}`
+
+- Importar o certificado assinado para o banco de dados dos solicitantes:
+
+`certutil -A -n "{{Server-cert}}" -t ",," -i {{path/to/file.crt}} -d .`
```

Comment:
is this a placeholder? we use `snake_case` for placeholders, also they should be translated
```suggestion
`certutil -A -n "{{server_certificate}}" -t ",," -i {{caminho/para/arquivo.crt}} -d .`
```